libs/js-sdk/src/client.ts

```diff
@@ -65,22 +59,22 @@ export function createBrowserClient({
 
     // Ensure that the init promise doesn't cause an unhandled promise rejection
     initialisedState.initialisedPromise.promise.catch(() => {})
-    const state = new EffectiveFeaturesState(
-        audiences,
-        store || new MemoryEffectiveFeatureStore(initialValues),
-    )
+
+    const stateStore = new EffectiveFeatureStateStore(audiences, initialValues)
 
     const updateStrategyImplementation = resolveUpdateStrategy(
         updateStrategy,
         environmentApiKey,
         api || featureBoardHostedService,
     )
 
-    debugLog('SDK connecting in background (%o)', {
-        audiences,
-    })
-    updateStrategyImplementation
-        .connect(state)
+    const retryCancellationToken = { cancel: false }
```

Comment:
FYI there is now a inbuilt type for this, https://developer.mozilla.org/en-US/docs/Web/API/AbortController